### ISSUE TITLE
Simplify email export for mailing list purposes

### ISSUE THREAD

**Issue by n8kim1** [battlecode/galaxy#500]
**note: this has the same ends / purpose as #554. Try that first, and if that fails, try this one instead.**

Right now you can click around the backend to get a full csv of users with all their info. It's nice.
But then to upload to mmblanche, you have to strip out the header and all the other info, and produce just one column of emails. This is doable with a csv editor, but it takes an annoying amount of manual effort every time. 
Simplify by having (an option for) the exported csv just having email addresses and no header if doable. Alternatively I guess an external python script works.

**Comment by j-mao** [comment]
Probably mutually exclusive with #554. Which do we prefer?

**Comment by n8kim1** [comment]
would strongly pref #554. although I'm not actually sure that one is _viable_, so let's leave this open as a fallback?

**Comment by j-mao** [comment]
On the contrary, I'd strongly pref this, because I'm very wary of tangling siarnaq up with MIT emailing systems.

**Comment by n8kim1** [comment]
This would have little value. Turns out, we don't use our competitor email lists much (and not at all in IAP 2024!), especially with other alternatives like Discord and the website, and we did fine. Also, the burden to tell competitors things isn't on us -- if competitors miss something, that's not our "fault" and not necessarily our "burden"/guilt/etc.
(I do wonder how many more competitors we'd have if we did email all the time too. But, we don't think there's many competitors that we're missing out on. Serious competitors would be checking announcements anyways. And there would be relatively few competitors who signed up and forgot would see their email, be reminded, and thus play fully.)

Also, this would be nontrivial code, and be code that specifically has a small niche too. Would be a lot of responsibility to create and maintain.
And, the entire workflow to build an email list is already a long process to do.

So choosing to not make improvements here, and work on better things instead. 
We can keep the previous systems around, but not sink time into improving them.

(we = me Serena Evan, who have worked w these lists a lot; also Maggie and Lowell heard and didn't object)
